tests/test_uds_auxiliary.py

```diff
@@ -327,5 +335,6 @@ def test_tester_present_sender(self, uds_raw_aux_inst, mocker):
             sleep(3)
 
         send_mock.assert_called_with(
-            UDSCommands.TesterPresent.TESTER_PRESENT_NO_RESPONSE
+            UDSCommands.TesterPresent.TESTER_PRESENT_NO_RESPONSE,
+            response_required=False,
```

Comment:
What about a TC where it is True?

Reply:
It is tested in other unit tests, but for the tester present sender it will always be False, as we do not expect an answer

Reply:
ok